### PR TITLE
Move the classess into their own files

### PR DESCRIPTION
**Pull request by matzipan**

### CODE REVIEW

**Comment by matzipan** [comment]
As @danrabbit  asked, I've removed the commit which removes the filemanager plugin for now.

File: plugins/folder-manager/folder-manager.plugin [hunk 1]
```diff
@@ -4,7 +4,7 @@ Loader=C
 IAge=2
 Name=Folder Manager
 Description=Basic folder manager with file browsing
-Icon=folder-saved-search
-Authors=Julien Spautz <spautz.julien@gmail.com>
-Copyright=Copyright © 2013 Scratch Developers
+Icon=system-file-manager
```
Comment (danrabbit) [inline plugins/folder-manager/folder-manager.plugin]: This looks like leftovers from another branch

File: plugins/folder-manager/Settings.vala [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Julien Spautz <spautz.julien@gmail.com>
+ */
```
Comment (danrabbit) [inline plugins/folder-manager/Settings.vala]: Doesn't seem to be related to moving classes to their own files

File: plugins/folder-manager/FolderManagerPlugin.vala [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Julien Spautz <spautz.julien@gmail.com>, Andrei-Costin Zisu <matzipan@gmail.com>
+ */
```
Comment (danrabbit) [inline plugins/folder-manager/FolderManagerPlugin.vala]: Doesn't seem to be related to moving classes to their own files

File: plugins/folder-manager/FileView.vala [hunk 4]
```diff
@@ -1,30 +1,28 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Julien Spautz <spautz.julien@gmail.com>, Andrei-Costin Zisu <matzipan@gmail.com>
+ */
```
Comment (danrabbit) [inline plugins/folder-manager/FileView.vala]: Can you do header changes in another branch please?

File: plugins/folder-manager/File.vala [hunk 5]
```diff
@@ -1,22 +1,21 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Julien Spautz <spautz.julien@gmail.com>, Andrei-Costin Zisu <matzipan@gmail.com>
+ */
```
Comment (danrabbit) [inline plugins/folder-manager/File.vala]: This doesn't seem to be related to moving classes to their own files

File: .gitignore [hunk 6]
```diff
@@ -1,3 +1,2 @@
 build
-../src/config.vala
```
Comment (danrabbit) [inline .gitignore]: Can you do this in another branch please?
